### DISCUSSION TITLE
Raise exceptions twice without hiding

### DISCUSSION THREAD

**Discussion by mikazz** [fastapi/fastapi#9329] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import uvicorn

from fastapi.exceptions import HTTPException
from fastapi import FastAPI

app = FastAPI()

@app.get("/info")
def info():
    try:
        x = 1/0
    except ZeroDivisionError as e:
        # in console:
        # INFO:     127.0.0.1:44212 - "GET /info HTTP/1.1" 400 Bad Request
        # and to the client:
        # Error: Bad Request {"detail": "division by zero"}
        raise HTTPException(status_code=400, detail=str(e))


if __name__ == '__main__':
    uvicorn.run(app, port=8000)
```

### Description

Hello

When I raise the HTTPException its only showing 400 Bad Request in my logs, the whole part with details is shown to client only, I wish I had complete exception stack trace in my logs. Im not sure if I should handle and raise again somehow in middleware?

### Operating System

Linux

### Operating System Details

Ubuntu

### FastAPI Version

0.94.1

### Python Version

Python 3.10.6

**Comment by y-mathur-i** [comment]
You can capture result of all the api calls in a middleware. Where you can capture the exception on a higher level and proceed however you like.
